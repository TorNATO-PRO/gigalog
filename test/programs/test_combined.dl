combined(X, Y, Z) :- Z = (X + Y) * (X - Y).
combined(5, 3, 16).
combined(10, 2, 96).
combined(7, 4, 33).
