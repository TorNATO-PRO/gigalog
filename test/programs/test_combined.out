combined(10,2,96)
combined(5,3,16)
combined(7,4,33)
